packages/validation/README.md

```diff
@@ -157,6 +157,19 @@ import { validateObosMembershipNumber } from '@obosbbl/validation/se';
 validateObosMembershipNumber('0000000') // => true
 ```
 
+### validateBankAccountNumber()
```

Comment:
Må du bestemme deg for hva metoden skal hete 😝

Reply:
hahah